workflow-templates/im-deploy-techdocs-site.yml

```diff
@@ -0,0 +1,66 @@
+# Workflow Code: SparklyToad_v1    DO NOT REMOVE
+# Purpose:
+#    Publishes a techdocs site to the storage account where TechHub
+#    looks for TechDocs.
+#
+# Frequency:
+#    - This workflow can be used once per TechDocs site or can be combined
+#      to publish multiple docs at once
+#
+# Projects to use this Template with:
+#    - Any project with TechDocs 
+
+name: Publish TechDocs Site
+
+# TODO:  Update the triggers where TechDocs will be published
+on:
+  # push:
+  # workflow_dispatch:
+  # pull_request:
+  #   types: [closed]
+
+jobs:
+  publish-techdocs-site:
+    runs-on: [self-hosted, im-techdocs]
```

Comment:
Is this `im-techdocs` runner going to be present in all of our organizations?
If so, is it a separate copy for each? If so, how do we keep them always the same?

Should there be a link here that lets the reader go see what MkDocs plugins get pre-installed on that runner?

I think there should also be instructions in this file about how to install additional MkDocs plugins.

Reply:
I think we would have a copy of this runner in each org.  I can talk to infra-purple about whether we can make this a "global" runner that gets copied to every org when there are changes, otherwise we would have to keep it up to date in each org.

Reply:
I chatted with Aric W and Eric J about having this as a "global" runner.  They indicated this work wasn't done yet but they didn't see a reason why it couldn't be a cross-org "global" type of runner.

I added some comments with a link to the im-practices version of the techdocs runner and what additional plugin installs look like.